app/services/imports/lettings_logs_field_import_service.rb

```diff
@@ -203,5 +205,30 @@ def update_reason(xml_doc)
         @logger.warn("lettings log with old id #{old_id} not found")
       end
     end
+
+    def update_homelessness(xml_doc)
+      return if meta_field_value(xml_doc, "form-name").include?("Sales")
+
+      old_id = meta_field_value(xml_doc, "document-id")
+      record = LettingsLog.find_by(old_id:)
+      if record.present?
+        return @logger.info("lettings log #{record.id} has a value for homeless and rp_homeless, skipping update") if record.rp_homeless == 1 && record.homeless.present?
+        return @logger.info("lettings log #{record.id} has a value for homeless and reasonpref is not yes, skipping update") if record.reasonpref != 1 && record.homeless.present?
+        return @logger.info("lettings log #{record.id} reimport values are not homeless - 11 and rp_homeless - yes, skipping update") if unsafe_string_as_integer(xml_doc, "Q14b1").blank? || unsafe_string_as_integer(xml_doc, "Q13") != 11
```

Comment:
this line says it checks if rp_homeless is yes but checks if it's blank instead - is that intentional?

Reply:
I'm guessing it's a 1 or nil situation?

Reply:
Yeah, basically if this field has anything in it we set rp_homeless to true/checked

Reply:
We would not set it if it's blank, we'd return